docs/upgrading-eck.asciidoc

```diff
@@ -47,6 +47,7 @@ CAUTION: These instructions are general guidelines only. You should have a thoro
 * Remove `setVmMaxMapCount`. See: <<{p}-virtual-memory>>.
 * Remove `groups` from `updateStrategy`. See: <<{p}-update-strategy>>.
 * Remove `featureFlags`
+* If you specify a `secureSettings`, convert the `secretName` to an array item
```

Comment:
If you specified `secureSettings`, convert `secretName` into an array item by prefixing it with `-`